admin/services/sas-token.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const azure = require('azure-storage')
+const moment = require('moment')
+
+require('dotenv').config()
+
+const addPermissions = azure.QueueUtilities.SharedAccessPermissions.ADD
+const storageConnection = process.env.AZURE_STORAGE_CONNECTION_STRING
+
+if (!storageConnection) {
+  throw new Error('An AZURE_STORAGE_CONNECTION_STRING is a required environment variable.')
+}
+const azureQueueService = azure.createQueueService(storageConnection)
```

Comment:
this is what is causing travis to fail, as it runs as soon as the app starts.  We need to explore whether we should do this once on start up, or each time it is used.  
We can add new encrypted vars next week to the travis config file.